assets/javascripts/discourse/templates/chat-channel-info.hbs

```diff
@@ -1,6 +1,11 @@
 {{compute-full-page-height selector=".channel-info"}}
 
 <div class="channel-info">
+  <LinkTo @route="chat.browse" class="channel-info__back-btn">
+    {{d-icon "chevron-left"}}
+    {{i18n "chat.channel_info.back-to-all-channels"}}
```

Comment:
I generally prefer to wrap this in a span, I have always found text with no direct parent to act weirdly.